libs/ha/ha_common_libs_k8s.py

```diff
@@ -208,6 +213,17 @@ def delete_s3_acc_buckets_objects(self, s3_data: dict, obj_crud: bool = False):
                         response = s3_del.delete_multiple_objects(_bucket, obj_list[1], quiet=True)
                         LOGGER.debug("Delete multiple objects response %s", response)
                 return True, "Successfully performed Objects Delete operation"
+            if is_ver:
+                try:
+                    obj_name = v_etag.pop("obj_name")
```

Comment:
Can't we get objects in bucket and version IDs if any here only, instead of passing all the data to method?

Reply:
> Can't we get objects in bucket and version IDs if any here only, instead of passing all the data to method?

As discussed, we will take care of this optimization in upcoming sprint.